pkg/workloads/spark_job/spark_job.py

```diff
@@ -237,7 +237,7 @@ def validate_transformers(spark, ctx, cols_to_transform, raw_df):
 
     logger.info("Sanity checking transformers against the first {} samples".format(TEST_DF_SIZE))
     sample_df = raw_df.limit(TEST_DF_SIZE).cache()
-    test_df = raw_df.limit(TEST_DF_SIZE).cache()
+    test_df = raw_df.limit(TEST_DF_SIZE)
```

Comment:
maybe this would be more clear?

```python
test_df = raw_df.limit(TEST_DF_SIZE).cache()
sample_df = test_df
```

Also what do you think of renaming `sample_df` to `transformed_df`?

Reply:
I think we want `test_df`and `sample_df` to be distinct objects